addons/web_editor/static/src/js/backend/html_field.xml

```diff
@@ -2,9 +2,11 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="web_editor.HtmlField" owl="1">
-        <t t-if="props.readonly || props.notEditable || (containsComplexHTML and !state.showCodeView)">
+        <t t-if="props.readonly || props.notEditable || (safeHtmlEdit and !state.showCodeView)">
             <t t-if="this.showIframe">
-                <iframe t-ref="iframe" t-att-class="{'d-none': !this.state.iframeVisible, 'o_readonly': true}"></iframe>
+                <iframe t-if="!safeHtmlEdit" t-ref="iframe" t-att-class="{'d-none': !this.state.iframeVisible, 'o_readonly': true}"></iframe>
+                <!--allow-same-origin and allow-scripts should remain mutually exclusive-->
```

Comment:
I'm confused by this comment. We don't seem to use `allow-scripts` anywhere anyway so what's the deal here? :p I feel like this might be an outdated comment ? Did I miss something ?

Reply:
Also in case anyone asks, we need 2 fields because if 'sandbox' is set with an empty value, it will just be sandboxed not allow anything. afaik there isn't a way to conditionally not set the attribute at all without xpath.

Reply:
In the old qweb I think there used to be something about `null` or `undefined` being treated differently than `false` for setting attributes. I think I remember that one of those values could end up not setting the attribute. Or maybe it was `None` vs `False` in the Python QWeb ? There used to be a trick like that, I think. Could you just have a try to be sure, please ? Obviously if it doesn't work then the duplication is fine.

Reply:
Oh nice, yeah null seems like it doesn't set the attribute at all